device/iot-device-samples/device-reconnection-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/DeviceClientManagerSample.java

```diff
@@ -71,8 +131,8 @@ private static void startSendingTelemetry() {
                 log.error("Exception thrown while sending telemetry: ", e);
             }
             try {
-                log.debug("Sleeping for {} secs before sending next message.", SLEEP_DURATION_IN_SECS);
-                Thread.sleep(SLEEP_DURATION_IN_SECS * 1000);
+                log.debug("Sleeping for {} secs before sending next message.", sleepTime);
+                Thread.sleep(sleepTime * 1000);
```

Comment:
You can change this to ```TimeUnit.SECONDS.sleep(sleepTime)``` to avoid needing to do this conversion from seconds to milliseconds

Reply:
Good idea, need to import another package to use that. Done.